customfiles/depenguin_bsdinstall.sh

```diff
@@ -26,6 +26,7 @@ fi
 # safe(r) separator for sed
 sep=$'\001'
 
+# shellcheck disable=SC1091 disable=SC2154 source-path=SCRIPTDIR source=depenguin_settings.sh.sample
```

Comment:
I think it’s better to change how shellcheck is invoked.

it’s easiest to figure this out locally and then adjust the GitHub action.

Reply:
should be fixed now

Reply:
Hm still unhappy, I don’t know too much about actions unfortunately 

```
./customfiles/depenguin_bsdinstall.sh:14:4: note: Not following: depenguin_settings.sh was not specified as input (see shellcheck -x). [SC1091]
```

Reply:
I think instead of specifying external-sources, you could just pass the specific file included in the action to shellsheck, then it will be considered

Reply:
ignoring problem checks seems to be the only way for now